host_applications/linux/apps/raspicam/RaspiVid.c

```diff
@@ -542,13 +544,13 @@ static void application_help_message(char *app_name)
       fprintf(stdout, ",%s", raw_output_fmt_map[i].mode);
    }
 
-   fprintf(stdout, "\n");
+   fprintf(stdout, "\n\n");
```

Comment:
Unrelated change ;-)

Reply:
Yep....noticed a formating problem while testing. I coudl send it in a different PR, but I have a horrible feeling the life is too short.

Reply:
:-)